src/utils/tree.js

```diff
@@ -13,29 +13,33 @@ var tf =
     },
 
     showTree: function(newickStr) {
-      var newick = this.require("biojs-io-newick");
+      var newick = require("biojs-io-newick");
```

Comment:
nope we can't do that. `this.require` is async` - it will always take the latest version ;-)
`require` will bundle the `msa-tnt` component (huge!,2MB uncomprossed) with the msa viewer.